i18n/locales/source/blockly-mooc/applab.json

```diff
@@ -433,6 +433,8 @@
   "stepIn": "Step in",
   "stepOver": "Step over",
   "stepOut": "Step out",
+  "submitYourProject": "Submit your project",
+  "submitYourProjectConfirm": "You cannot edit your project after submitting it, really submit?",
```

Comment:
You should add either add them to apps/i18n/applab/en_us.json or apps/i18n/common/en_us.json.